numpy/linalg/linalg.py

```diff
@@ -1382,7 +1382,7 @@ def cond(x, p=None):
 
     Parameters
     ----------
-    x : (M, N) array_like
+    x : (..., M, N) array_like
```

Comment:
Google it ;), I think it is well defined with the pseudo inverse as well, since it has to do with how well the solution of the system of linear equations are, which also has a well defined solution for non-square matrices. At least in a least square sense.